### PR TITLE
fix: depend link priority on linktype

### PR DESCRIPTION
**Pull request by jeroenreijs**
also change some labels

### CODE REVIEW

File: src/components/button.js [hunk 1]
```diff
@@ -8,9 +8,19 @@
...
+          linkToExternal,
+          disabled,
+          buttonText,
+        } = options;
 
-        if (linkToExternal && linkToExternal !== '') {
+        if (
+          linkType === 'External' &&
+          linkToExternal &&
+          linkToExternal !== ''
```
Comment (kupara) [inline src/components/button.js]: I am curious why we have both this check and the one above it. Are there cases where `linkType` is external and `linkToExternal` is undefined/null?

**Comment by kupara** [comment]
Also are we not adding this to the other prefabs as well eg.g navigation items

File: src/components/button.js (hunk 1 above)
Comment (jeroenreijs) [reply src/components/button.js]: Good point, as the default value is an empty string, I assume the value can never be undefined.
I will check it and probably change it.

**Comment by jeroenreijs** [comment]
Adding this functionality to other prefabs is not part of this ticket.

File: src/components/button.js (hunk 1 above)
Comment (kupara) [reply src/components/button.js]: aight

**Pull request merged**